### PR TITLE
AAU, on the CLI, I see a better structure in the code

### PR DESCRIPTION
**Pull request by FlorianLeRoyKili**
- [ ] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request and forced security tests.
- [ ] I opened a pull request on the Node playground if the changes are breaking.

### CODE REVIEW

File: kili/cli/cli.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+@cli.group(context_settings=CONTEXT_SETTINGS)
+def project():
+    """Commands to interact with a Kili project"""
+
+
+project.add_command(create, name="create")
+project.add_command(describe, name="describe")
+project.add_command(import_assets, name="import")
```
Comment (PierreLeveau) [inline kili/cli/cli.py:31]: why not renaming the python functions into the command names? this would be even clearer.

**Review by PierreLeveau — CHANGES_REQUESTED** [review]
2 suggestions

File: kili/cli/cli.py (hunk 1 above)
Comment (FlorianLeRoyKili) [reply kili/cli/cli.py:31]: You mean : `from kili.cli.project.create import create_project` then `project.add_command(create_project, name="create")`. If so, then I agree.

File: kili/cli/cli.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+@click.version_option(__version__)
+def cli():
+    """Kili Command line Interface
+
+       To get all the available commands, please type: `kili project --help`.
+    """
+
+
+@cli.group(context_settings=CONTEXT_SETTINGS)
+def project():
```
[the remark this answers is not in the document]
Comment (FlorianLeRoyKili) [reply kili/cli/cli.py]: Apparently use add_command also works with group so we're OK !

File: kili/cli/cli.py (hunk 1 above)
Comment (PierreLeveau) [reply kili/cli/cli.py]:
```
from kili.cli import project
project.add_command(project.create, name="create")
```

**Comment by theodu** [comment]
@FlorianLeRoyKili you miss the __init__ in the CLI folder. I already had this problem where I forget to add a __init__ in a new folder, it was working in local but not when packaged.

**Review by theodu — CHANGES_REQUESTED** [review]
@FlorianLeRoyKili you miss the init in the CLI folder. I already had this problem where I forget to add an init in a new folder, it was working in local but not when packaged.

**Review by theodu — CHANGES_REQUESTED** [review]

**Comment by PierreLeveau** [comment]
Linting issue, some you can ignore I think

**Review by theodu — APPROVED** [review]

**Pull request merged**